src/report.php

```diff
@@ -0,0 +1,66 @@
+<?php
+    include("includes/header.php");
+    include("includes/nav.php");
+?>
+
+<main role="main" class="col-md-9 ml-sm-auto col-lg-10 px-md-4">
+  <div class="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pt-3 pb-2 mb-3 border-bottom">
+    <h1 class="h2">
+      Report
+    </h1>
+    <div class="btn-group mr-2">
+    <input type="submit" class="btn btn-success" value="GENERATE" name="GENERATE" id="report">
+    </div>
+  </div>
+
+  <div id="error"></div>
+
+  <form method="GET" action="report.php">
+    <div class="form-group mb-2">
+        <div class="row">
+            <div class="col-sm">
+                <label for="recorddatestart">Start Date </label><input class="form-control mr-sm-2" type="date" id="recorddatestart" name="recorddatestart" value="2020-09-04" required>
+            </div>
+
+            <div class="col">
+            <label for="recorddateend">End Date </label><input class="form-control mr-sm-2" type="date" id="recorddateend" name="recorddateend" value="2020-10-16" required>
+            </div>
+            <div class="col">
+                <input type="radio" class="form-check-input" id="category" name="WHICHDATA" value="CATEGORY"><label for="category">Catergory </label><input class="form-control mr-sm-2" type="text" id="CATEGORYID" placeholder="Category Name"></p>
```

Comment:
Rename to category ID/implement category ID lookup

Reply:
Just added more clarification on the label + the placeholder.